lib/dictionary.rb

```diff
@@ -0,0 +1,26 @@
+require "./lib/scrabble.rb"
+require "./lib/player.rb"
+
+module Scrabble
+  class Dictionary
+    attr_reader :language, :file
+
+    def initialize(language = :english)
+      @language = language
+    end
+
+    def check_word(word)
+      dict_file = File.open("./lib/puzzler_list.txt", "r")
```

Comment:
This would be an excellent place to use the `||=` operator that we learned about last week, possibly with an instance variable.